pom.xml

```diff
@@ -285,6 +286,17 @@
             <resource>
```

Comment:
Let's add a detailed description here, explaining the problem this exclusion / inclusion comes to solve.